keras/layers/convolutional.py

```diff
@@ -379,6 +379,95 @@ def get_config(self):
         return dict(list(base_config.items()) + list(config.items()))
 
 
+class TransposedConvolution2D(Convolution2D):
```

Comment:
I'd rather use `Deconvolution2D` (and an alias `Deconv2D`), to better match the TensorFlow API.